### ISSUE TITLE
forceSyncFilters broken

### ISSUE THREAD

**Issue by b4545** [honestbleeps/Reddit-Enhancement-Suite#5177]
What's up?

I have added a short list of subreddits to the filteReddit module and would like to have those subreddits synced to my native /r/all filters, as described in useRedditFilters and forceSyncFilters.

I have found that there is no automatic syncing taking place. I also get an error when I do a force sync:

https://www.reddit.com/api/filter/user/[redacted]/f/all?app=res failed with status 404
Where does it happen?

In the filteReddit module in RES settings useRedditFilters and forceSyncFilters.

Screenshots or mock-ups

Settings: https://i.imgur.com/PvAMSkR.png

Error: https://i.imgur.com/GL1xiye.png

Browser console: https://i.imgur.com/LlypROs.png

What browser extensions are installed?

All extensions except RES turned off.

Night mode: true
RES Version: 5.18.8
Browser: Chrome
Browser Version: 79
Cookies Enabled: true
Reddit beta: false

**Comment by BenMcGarry** [comment]
I am unable to repo this on 5.18.8. My force sync is working fine as well as automatic sync. Are you able to message /r/resissues with the username you are trying? It's hard to repo this without knowing the existing account incase theres odd characters breaking it.

**Comment by b4545** [comment]
This is my original post from RESissues, and I'm posting here because it received no help for several days there. The user name is 20 chars a-z 0-9.

Only help from RESissues:

> You appear to be using a beta/nightly version of Chrome. Some issues, particularly visual glitches, are usually caused by browser bugs. The issue may resolve itself in a day or two. Would you like to know more, citizen?
> 
> I am a bot, and this action was performed automatically. Please contact the moderators of this subreddit if you have any questions or concerns.

**Comment by b4545** [comment]
I also did a factory reset in RES, reopened the browser, and tried it again and received the same error;

![image](https://user-images.githubusercontent.com/21189387/71645752-87081780-2ca2-11ea-9be7-a44c04092ce7.png)

**Comment by b4545** [comment]
I was able to get it to work, but I think you have a bug. I started with empty /r/all filter list. I added a filteReddit filter for the first subreddit, then went to the settings and saw it in the RES filters. I tried to sync that filter and received the 404 error. Then I went to frontpage and manually added that filter to the native filter list. Then I used filteReddit to filter another subreddit. Again, went to settings and tried to sync, and this time it worked.
